File: lab/python/fatorial/fatorial2.py
def fatorial(n):
    resultado = 1
    expressao = []
    for i in range(n, 0, -1):   # vai de n até 1
        resultado *= i
        expressao.append(str(i))
    print(" x ".join(expressao), "=", resultado)
    return resultado


fatorial(5)

File: lab/javascript/fatorial/fatorial2.js
function fatorial(n) {
    let result = 1;
    let expression = '';

    for (let i = n; i > 1; i--) {
        result *= i;
        expression += i + ' x ';
    }
    expression += '1 = ' + result;
    return expression;
}

console.log(fatorial(5));
